app/assets/javascripts/admin/templates/components/dashboard-problems.hbs

```diff
@@ -18,7 +18,7 @@
         </div>
 
         <p class="actions">
-          {{d-button action=(action "refreshProblems") class="btn-default" icon="sync" label="admin.dashboard.refresh_problems"}}
+          {{d-button action=refreshProblems class="btn-default" icon="sync" label="admin.dashboard.refresh_problems"}}
```

Comment:
Is this change related to partials?

Reply:
I think yes ? AFAIK variables in context are passed to partials automatically, but as it's converted to a component it's not the case anymore, so I'm create the action when I call the component {{dashboard-problems}} which is in the context of having the "refreshProblems" action handler.

Or am I missing your point ?

Reply:
Ah, yes, it's an arg now. Carry on. 🙂

Btw. we should start using named `@args` (https://github.com/emberjs/rfcs/blob/e3d18e6206546f1c846ce593f2314ab3f536d253/text/0276-named-args.md) from what I can tell they've been around since Ember 3.0.0-beta1 😃 (that and `this.` for properties)

Reply:
indeed, do you want to do a post on dev about it please?